site/docs/v1/tech/apis/reactor.adoc

```diff
@@ -0,0 +1,124 @@
+---
+layout: doc
+title: Reactor APIs
+description: APIs for activating, deactivating, and retrieving the status of a license.
+---
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.26.0
+====
+
+This page contains the APIs for Reactor, FusionAuth's license system. Reactor is used to activate features based upon your licensing tier. Here are the APIs:
+
+* <<Activate Reactor license>>
+* <<Retrieve Reactor status>>
+* <<Regenerate Reactor license>>
+* <<Deactivate Reactor license>>
+
+
+== Activate Reactor license
+
+This API is used to activate a Reactor license.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Activate a Reactor license
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/reactor#
+--
+
+==== Request Body
+
+[.api]
+[field]#licenseId# [type]#[String]# [required]#Required#::
+The license Id to to activate.
```

Comment:
typo. I always try to cut and paste docs into google docs and run the spelling/grammar check to catch stuff like this :)

Reply:
That is a nice pro tip! Thanks